compiler/ast/src/ast_gen.rs

```diff
@@ -283,7 +283,7 @@ pub enum ExprKind<U = ()> {
 }
 pub type Expr<U = ()> = Located<ExprKind<U>, U>;
 
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Copy, Clone, Debug, PartialEq)]
```

Comment:
This change doesn't match to current version of generation script

Reply:
🤦